runtime/test_artifacts/generator.go

```diff
@@ -23,6 +23,12 @@ import (
 //  It might be an inevitability to have shared testing code, but would like more eyes on it.
 //  Look for opportunities to make testing completely modular
 
+const (
```

Comment:
+1 to moving this to `runtime/defaults`

Reply:
https://github.com/pokt-network/pocket/pull/177/commits/aa1c7378b3970831c19cb589a496d1ac4d499818